build.py

```diff
@@ -65,13 +65,13 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.13.0dev':
-      ('21.08dev',   # triton container
-       '21.06',      # upstream container
-       '1.8.0',      # ORT
-       '2021.2.200', # ORT OpenVINO
-       '2021.2',     # Standalone OpenVINO
-       '2.2.8')      # DCGM version
+    '2.13.0dev': (
+        '21.08dev',  # triton container
```

Comment:
The target branch should be r21.07 as I understand
It is ok for us to move to 2.2.8 for 21.08, we need this change in 21.07. 
@deadeyegoodwin feel free to correct me

Reply:
The revert to 2.2.3 and re-application of the patch is only needed on 21.07. We can keep the main branch as it is (i.e. with 2.2.8 and with no patch)